dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/task/sql/SqlParameters.java

```diff
@@ -251,8 +251,9 @@ public void dealOutParam(String result) {
                 sqlResultFormat.put(key, new ArrayList<>());
             }
             for (Map<String, String> info : sqlResult) {
-                for (String key : info.keySet()) {
-                    sqlResultFormat.get(key).add(String.valueOf(info.get(key)));
+                for (Map.Entry<String, String> entry : info.entrySet()) {
+                    String key = entry.getKey();
+                    sqlResultFormat.get(key).add(String.valueOf(entry.getValue()));
```

Comment:
Can we remove the String.valueOf() here? and it seems we use forEach is clearer
```java
info.foreach((key, value) -> {
sqlResultFormat.get(key).add(value);
})
```

Reply:
+ 1